_docs/compliance/pentest.md

```diff
@@ -18,4 +18,14 @@ If you, or a third party acting on your behalf, plan to perform a penetration te
 
 This notification is only necessary for in-depth security testing or significant load-testing, which is a common step in agency ATO processes for customer systems and in the software development lifecycle. You don't need an approval, and cloud.gov doesn't provide approvals. Simply sending the notification is sufficient. You can always run routine automated vulnerability scans on your own applications without special notification.
 
-When arranging a security assessment or penetration test, the system under test is _your_ application at _application-name_.app.cloud.gov, not the cloud.gov infrastructure -- as cloud.gov, and third-party assessors, test those continually and [make the results available through FedRAMP]({{ site.baseurl }}/docs/overview/fedramp-tracker/#start-the-ato-process). Your assessment must not target other cloud.gov customers, nor perform or simulate denial of service attacks or otherwise violate the [Amazon AWS testing policy](https://aws.amazon.com/security/penetration-testing/).
+When arranging a security assessment or penetration test, the system under test is one of:
+
+* _your_ application at _application-name_.app.cloud.gov, for cloud.gov Platform, or
```

Comment:
What about their external domain? Is there a reason they wouldn't be allowed to test against that?